src/prognostic_equations/zero_tendency.jl

```diff
@@ -0,0 +1,37 @@
+###
+### Zero grid-scale or subgrid-scale tendencies
+###
+
+function zero_tendency!(Yₜ, Y, p, t, colidx, ::NoGridScaleTendency, _)
+    # turn off all grid-scale tendencies
+    @. Yₜ.c.ρ[colidx] = 0
+    @. Yₜ.c.uₕ[colidx] = C12(0, 0)
+    @. Yₜ.f.u₃[colidx] = C3(0)
+    @. Yₜ.c.ρe_tot[colidx] = 0
+    for ρχ_name in filter(is_tracer_var, propertynames(Y.c))
+        @. Yₜ.c.:($$ρχ_name)[colidx] = 0
+    end
+    return nothing
+end
+
+function zero_tendency!(
+    Yₜ,
+    Y,
+    p,
+    t,
+    colidx,
+    ::NoSubgridScaleTendency,
+    ::PrognosticEDMFX,
+)
+    # turn off all subgrid-scale tendencies
+    n = n_prognostic_mass_flux_subdomains(p.atmos.turbconv_model)
+    Yₜ.c.sgsʲs[colidx] .= zero(eltype(Yₜ.c.sgsʲs[colidx]))
+    Yₜ.f.sgsʲs[colidx] .= zero(eltype(Yₜ.f.sgsʲs[colidx]))
+    # for j in 1:n
```

Comment:
Is this still needed?

Reply:
This is testing gpu. I’ll see what works.